.pipelines/pipeline.user.windows.yml

```
@@ -0,0 +1,115 @@
+      logs:
+        - from: 'WinSdkInstall'
+          to: 'WinSdkInstall'
+          include:
+            - '**/*.log'
+    - !!defaultcommand
+      name: 'Install Java 7 JRE'
+      command: '.scripts/installJava7jre.cmd'
+    - !!defaultcommand
+      name: 'Install Java 8 JDK'
+      command: '.scripts/installJava8jdk.cmd'
+    - !!defaultcommand
+      name: 'Clean and Resolve Dependencies'
+      command: '.scripts/gradle.cmd'
+      arguments: 'clean resolveDependencies --refresh-dependencies'
+    - !!defaultcommand
+      name: 'List Installation Information'
+      command: '.scripts/dumpInstallationInfo.cmd'
+test:
+  commands:
+    # - !!defaultcommand
+    #   name: 'Gradle Static Analysis'
+    #   command: '.scripts/gradle.cmd'
+    #   arguments: 'check -x test'
+    #   logs:
+    #     - to: 'Checks Reports'
+    #       include:
+    #         - '**/build/reports/**/*'
+    #       exclude:
+    #         - '**/build/reports/tests/**/*'
+    - !!testcommand
+      name: 'ApplicationInsights-Java Unit Tests'
+      command: '.scripts/gradle.cmd'
+      arguments: 'test'
+      fail_on_stderr: false
+      testresults:
+        - title: ':core Unit Test Results'
+          type: 'junit'
+          from: 'core/build/test-results/text'
+          include:
+            - '**/*'
+        - title: ':agent Unit Test Results'
+          type: 'junit'
+          from: 'agent/build/test-results/test'
+          include:
+            - '**/*'
+        - title: ':ApplciationInsightsInternalLogger Unit Test Results'
+          type: 'junit'
+          from: 'ApplciationInsightsInternalLogger/build/test-results/test'
+          include:
+            - '**/*'
+        - title: ':azure-application-insights-spring-boot-starter Unit Test Results'
+          type: 'junit'
+          from: 'azure-application-insights-spring-boot-starter/build/test-results/test'
+          include:
+            - '**/*'
+        - title: ':web Unit Test Results'
+          type: 'junit'
+          from: 'web/build/test-results/test'
+          include:
+            - '**/*'
+        - title: ':web-auto Unit Test Results'
+          type: 'junit'
+          from: 'web-auto/build/test-results/test'
+          include:
+            - '**/*'
+        - title: ':logging:common Unit Test Results'
+          type: 'junit'
+          from: 'logging/common/build/test-results/test'
+          include:
+            - '**/*'
+        - title: ':logging:log4j1_2 Unit Test Results'
+          type: 'junit'
+          from: 'logging/log4j1_2/build/test-results/test'
+          include:
+            - '**/*'
+        - title: ':logging:log4j2 Unit Test Results'
+          type: 'junit'
+          from: 'logging/log4j2/build/test-results/test'
+          include:
+            - '**/*'
+        - title: ':logging:logback Unit Test Results'
+          type: 'junit'
+          from: 'logging/logback/build/test-results/test'
+          include:
+            - '**/*'

```

can we do same as 3.0 build?
```suggestion
        - title: ':Unit Test Results'
          type: 'junit'
          include:
            - '**/build/test-results/test/**/*'
```